### ISSUE TITLE
Add new edematous anatomical entity pattern template

### ISSUE THREAD

**Issue by rays22** [obophenotype/upheno#805]
1. [x] add new pattern `edematousAnatomicalEntity.yaml` for review
2. [x] get feedback
3. [ ] fix or improve pattern based on feedback

* Examples:
	1. [XPO:0141324	edema in brain](https://www.ebi.ac.uk/ols/ontologies/xpo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FXPO_0141324)
	2. [XPO:0141593	edema in lung](https://www.ebi.ac.uk/ols/ontologies/xpo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FXPO_0141593)
	3. [XPO:0115698	abnormally edematous pericardium](https://www.ebi.ac.uk/ols/ontologies/xpo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FXPO_0115698)

**Comment by ybradford** [comment]
this looks good to me. ZP uses edematous a lot for example: http://purl.obolibrary.org/obo/ZP_0000038 and http://purl.obolibrary.org/obo/ZP_0001451 

It seems the ZP is already using characteristic_of which differs from XPO http://purl.obolibrary.org /obo/XPO_0141324

Orcid id 0000-0002-9900-7880

**Comment by sbello** [comment]
http://www.informatics.jax.org/vocab/mp_ontology/MP:0001785

**Comment by srobb1** [comment]
I definitely want to understand the implications and differences between inheres_in and characteristic_of. I use this template in PLANP and want to make sure it still works for us.

https://www.ebi.ac.uk/ols/ontologies/planp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPLANP_0000801
https://www.ebi.ac.uk/ols/ontologies/planp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPLANP_0000802
https://www.ebi.ac.uk/ols/ontologies/planp/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPLANP_0000803

ORCID 0000-0002-3528-5267

**Comment by rays22** [comment]
> 0000-0002-3528-5267

@srobb1 ,
There is no difference between `inheres in` and `characteristic of` in the sense that they both refer to [RO:0000052.](https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FRO_0000052). `characteristic of` is the current label of RO:0000052. There is an explanation why the label was changed [in the comment section of RO:0000052 here.](https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FRO_0000052)
